### ISSUE TITLE
Web Protection and Timeline included?

### ISSUE THREAD

**Issue by AlexJPawlak** [MicrosoftDocs/microsoft-365-docs#7364]
Hello! 
Comparison charts dont show that clearly, but is device timeline and web protection included? I have onboarded Defender for Business, yet Device Timeline is missing, and Web Protection reports shows as if I'm missing correct license. (https://security.microsoft.com/webprotection )  whereas accessing https://security.microsoft.com/machines/machineID/timeline redirects me to homepage . Could that be clarified which features are not there in Defender for Business subscription?

Web Protection set to 'block' mode does work, and websites are blocked, but we have no visibility on the blocked website events - quite impaired experience
Thanks! 

Alex Pawlak

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 587ea70a-04b9-bdc8-f774-419607a07c47
* Version Independent ID: 587ea70a-04b9-bdc8-f774-419607a07c47
* Content: [Compare Microsoft Defender for Business to other Microsoft 365 plans](https://docs.microsoft.com/en-us/microsoft-365/security/defender-business/compare-mdb-m365-plans?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-business/compare-mdb-m365-plans.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-business/compare-mdb-m365-plans.md)
* Product: **m365-security**
* Technology: **mdb**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

**Comment by denisebmsft** [comment]
Hello @AlexJPawlak and thank you for your questions. I am working on getting the answers for you.

**Comment by AlexJPawlak** [comment]
Thanks Denise! I have also filed a feedback from the portal, suggesting even a limited timeline of 7 days - this could help customers trace events on the device and draw some reports from device activity or pinpoint false positives that can be actioned / exceptioned

**Comment by denisebmsft** [comment]
Hello @AlexJPawlak. Thanks again for the feedback, both here and in the portal. I have confirmed that web protection capabilities are included in Defender for Business, but not the device timeline view or detailed reports about blocked websites. And I have passed your feedback along to the product team as well. As this isn't necessarily a documentation issue, I am going to close it at this time, but I did want you to know we appreciate the feedback very much.
